app/Http/Controllers/PageController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+    namespace App\Http\Controllers;
+    use App\Http\Controllers\Controller;
```

Comment:
This is also not needed. Please remove. The controller it inherits is already in the same namespace